internal/storage/storage_handle.go

```diff
@@ -77,6 +94,18 @@ func NewStorageHandle(ctx context.Context,
 		}),
 		storage.WithPolicy(storage.RetryAlways))
 
-	sh = &storageClient{client: sc}
+	client := &storageClient{client: sc}
+	return client, err
```

Comment:
simple return should be sufficient. no need to mention client, err

Reply:
Removed the client variable.